python/mlx/optimizers.py

```diff
@@ -265,3 +264,54 @@ def apply_single(
         state["v"] = v
 
         return parameter - lr * gradient / (mx.sqrt(v) + eps)
+
+ 
+class RMSprop(Optimizer):
+    r"""Implementation of the RMSprop optimizer.
+
+    The RMSprop update adjusts the Adagrad method in a very simple way in an
+    attempt to reduce its aggressive, monotonically decreasing learning rate.
+    Specifically, it uses a moving average of squared gradients to normalize
+    the gradient itself. That has an effect of balancing the step size decrease
+    (equivalently, divide the gradient by the root of the mean of the squares
+    of the gradients):
+
+    .. math::
+
+        v_{t+1} &= \beta v_t + (1 - \beta) g_t^2 \\
+        w_{t+1} &= w_t - \frac{\lambda}{\sqrt{v_{t+1} + \epsilon}} g_t
+
+    Where:
+    - :math:`\beta` is the decay rate that controls the moving average over past squared gradients.
```

Comment:
The comment mentions `beta` and the class member is `alpha`. Could you maybe choose one of the two?